scripts/integration/docker-compose.http-scrape.yml

```diff
@@ -0,0 +1,75 @@
+version: "3"
+
+services:
+  # https://github.com/sigoden/dufs
+  # serves static files at an HTTP endpoint
+  dufs:
+    image: docker.io/sigoden/dufs:latest
+    networks:
+      - backend
+    command:
+      - "/data"
+    volumes:
+      - ${PWD}/tests/data/http-scrape/:/data
+  # the project suggests that two services should not be necessary
+  # yet I was not able to only auth protect one path for view/download access.
```

Comment:
It's _probably_ fine since we're already starting additional containers for HTTPS as well - but 🤷

Reply:
yeah, I can probably remove this comment. It was more to say "I tried" to use just one. But I guess not a huge deal either as well since its all using the same image.

Reply:
updated the comments in that file in https://github.com/vectordotdev/vector/pull/13793/commits/32acc20f385ee0dcfbd3f122b51a44dd4ba793f8